Arrowgene.Ddon.GameServer/Characters/ExpManager.cs

```diff
@@ -391,15 +391,15 @@ public static CDataCharacterJobData CalculateBaseStats(CharacterCommon Character
             return JobData;
         }
 
-        public void AddExp(GameClient client, CharacterCommon characterToAddExpTo, uint gainedExp, byte type = 0)
+        public void AddExp(GameLogicSetting gameSettings, GameClient client, CharacterCommon characterToAddExpTo, uint gainedExp, byte type = 0)
```

Comment:
I'm not sure about this, since ExpManager is part of DdonGameServer, couldnt GameLogicSetting be fed on construction?

Reply:
Good point, I was thinking that. I was also wondering if maybe something like this should just be available via a singleton.